api/spotify.py

```diff
@@ -111,6 +117,9 @@ def makeSVG(data, background_color, border_color):
     barCount = 84
     contentBar = "".join(["<div class='bar'></div>" for _ in range(barCount)])
     barCSS = barGen(barCount)
+    barPalette = gradientGen(data["item"]["album"]["images"][1]["url"], 4)
```

Comment:
These two lines would error out when the user isn't actively listening to anything.
Probably best to make use of the two checks below (if playing + if album art exists)

```
[ERROR] Exception on /api/spotify [GET]
Traceback (most recent call last):
  File "/var/task/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/var/task/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/var/task/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/var/task/flask/_compat.py", line 39, in reraise
    raise value
  File "/var/task/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/var/task/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "./api/spotify.py", line 175, in catch_all
    svg = makeSVG(data, background_color, border_color)
  File "./api/spotify.py", line 120, in makeSVG
    barPalette = gradientGen(data["item"]["album"]["images"][1]["url"], 4)
KeyError: 'item'
```

Reply:
Fixed, sorry about that one, I noticed it today

Reply:
I added a PLACEHOLDER_URL as a band-aid to that, I completely overlooked those two checks :eyes: